collections/Douce/MS_Douce_311.xml

```diff
@@ -30,61 +30,559 @@
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 121r–v)</locus>
+                        <title type="desc">Short prayers</title>
+                        <textLang mainLang="la" otherLangs="frm">Rubrics in Middle French</textLang>
+                     </msItem>
+                     <msItem>
+                        <title type="desc">Prayers to Christ</title>
+                        <msItem>
+                           <locus>(fol. 122r)</locus>
+                           <note>Blank</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 122v–124r)</locus>
+                           <title type="desc">Prayer to Christ</title>
+                           <textLang mainLang="frm">Middle French</textLang>
+                        </msItem>
+                        <msItem>
+                           <locus>(fol. 124a verso)</locus>
+                           <title type="desc">Prayer to Christ</title>
+                           <textLang mainLang="la">Latin</textLang>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 124b verso-124c verso)</locus>
+                           <note>Blank, ruled</note>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <title type="desc">Office of the dead</title>
+                        <msItem>
+                           <locus>(fol. 125r)</locus>
+                           <note>Blank</note>
+                        </msItem>
+                        <msItem>
+                           <locus>(fols. 125v–145r)</locus>
+                           <title key="work_13574" type="desc">Office of the dead, Use of Rome</title>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 145v)</locus>
+                        <note>Blank, ruled</note>
+                     </msItem>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent>
+                           <measure>ii (modern flyleaves) + 150 + ii (modern flyleaves) leaves</measure>
+                           <dimensions type="leaf" unit="mm">
+                              <height>238</height>
+                              <width>178</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation (i–ii, 1–147), replacing a brown ink foliation (1–171) from the late seventeenth or eighteenth century, made before the removal of the miniatures and trimmed during rebinding.</foliation>
```

Comment:
is the correct, as there are references to 124a and 124b in the description (extent may also need to be corrected)